boards/arm/sam4l_ek/board.cmake

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2020 Gerson Fernando Budke <nandojve@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=atsam4lc4c")
```

Comment:
Can you have these properties added in a single line?

Reply:
The line will pass 80 char column and I prefer keep this way because of that.